### ISSUE TITLE
Add property taxes

### ISSUE THREAD

**Issue by rmx90210** [CityOfPhiladelphia/property2#162]
There has been a lot of demand coming in from users for property tax balanaces/taxes paid. Here's one request with some good context:

> It would be really helpful if this site disclosed the current year's billed real estate tax amount. Often time we have to call revenue to get this and were just told this information will no longer be provided to title companies. It is imperative we have this information to handle prorations to new owners. This is unacceptable. By adding this information to the website you would likely cut in half the amount of calls the call center receives. Please take this into consideration this is not private information and needs to be made available to real estate professionals so we can insure correct amounts are collected and the city gets it's tax payments. [...]

It seems like a no-brainer to try to get this information in Property Search. @timwis that data is already published in Socrata, right?

**Comment by timwis** [comment]
Yes - [the code has been written](https://github.com/CityOfPhiladelphia/property2/tree/taxbalance2) but the open data that's available doesn't indicate the balance if the balance is under payment agreement, so it's not ready yet. There should be an update on this next week or the following - was just talking to them about it a couple days  ago.
